packages/editor/src/components/collab-sidebar/add-comment.js

```diff
@@ -43,11 +51,31 @@ export function AddComment( {
 	}
 
 	return (
+		// Disable reason: role="listitem" does in fact support aria-expanded.
+		// eslint-disable-next-line jsx-a11y/role-supports-aria-props
 		<VStack
-			className="editor-collab-sidebar-panel__thread is-selected"
+			className={ clsx(
+				'editor-collab-sidebar-panel__thread is-selected',
+				{
+					'is-floating': isFloating,
+				}
+			) }
 			spacing="3"
 			tabIndex={ 0 }
 			role="listitem"
+			ref={ isFloating ? refs.setFloating : undefined }
+			style={
+				isFloating
+					? // Delay showing the floating note box until a Y position is known to prevent blink.
+					  { top: y, opacity: ! y ? 0 : undefined }
```

Comment:
Bit hacky, but fixes position blink - https://github.com/WordPress/gutenberg/pull/72494#issuecomment-3441662842.

Reply:
I tried, but it messed up positioning and focus transfer. The `null` can't have references from floating UI, so we end up with a stale `y` value.

With this method, we at least get the correct position when it's available.

Reply:
Actually, that probably won't work since we need the component to render to get its height which is used to calculate the y values.